Documentation/Tutorials/EdgeTutorialAppStart/EdgeTutorialAppStart/AppDelegate.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+/* Edge Tutorial - code section (1/3)
+import AEPAssurance
+import AEPCore
+import AEPEdge
+import AEPEdgeConsent
+import AEPEdgeIdentity
+import AEPLifecycle
+// Edge Tutorial - code section (1/3) */
```

Comment:
Would this app be used for other tutorials as well? If so, we can rename the sections accordingly to accommodate for that, for example we can have all comments like:
```
// Edge Tutorial 1 - code section (n/m)
```
and in the table of contents we refer to sendEvent as tutorial 1.

Reply:
I was thinking the number of tutorials in Edge. In case we have tutorial 2 and 3 in the future and use the same app, the steps may be conflicting with tutorial 1. 
But if the same app code will be used in other repos, might be confusing though. 
We can keep it as is for now and we can decide when new tutorials are added.